pkg/dispatchcli/cmd/update_api.go

```diff
@@ -17,8 +15,8 @@ import (
 )
 
 var (
-	updateAPILong    = "update an api based on json"
-	updateAPIExample = `dispatch update api my_api --path /new/path`
+	updateAPILong    = "update a name api "
```

Comment:
shouldn't this be "update an API by name"?  Also, are we being consistent with the phrasing across the CLI?